yarn-project/aztec.js/src/contract_deployer/deploy_sent_tx.ts

```diff
@@ -2,8 +2,11 @@ import { FieldsOf } from '@aztec/circuits.js';
 import { ContractAbi } from '@aztec/foundation/abi';
 import { TxHash, TxReceipt } from '@aztec/types';
 
+
```

Comment:
Not sure why there are additional space in here, would expect it to be handled by prettier.